### ISSUE TITLE
Default element for dots and navigation not interactive/focusable

### ISSUE THREAD

**Issue by squalrus** [OwlCarousel2/OwlCarousel2#1920]
Keyboard users cannot navigate to the dots or nav items with the default rendering of controls.

### Test Environment:
Windows 10 Chrome 58.0.3029.110

### Repro steps:
1. Navigate to https://owlcarousel2.github.io/OwlCarousel2/demos/basic.html
2. Tab through content until the carousel is reached.

### Actual Result:
Nav elements and dot elements cannot be selected.

### Expected Result:
Focus should be able to reach both dots and nav controls.

### Note:
Relates to https://github.com/OwlCarousel2/OwlCarousel2/issues/1717 and https://github.com/OwlCarousel2/OwlCarousel2/issues/110

**Comment by squalrus** [comment]
We implemented a solution using the API. Our implementation also needed to account for localization so we needed another layer of abstraction for handling that text.

```js
// Get text from a data-attr
prevText = $this.data('prev-text');
nextText = $this.data('next-text');

// Update 
navElement: 'button role="presentation"',
navText: [
    '<div aria-label="' + prevText + '">&#x2039</div>',
    '<div aria-label="' + nextText + '">&#x203a</div>'
]
```
